src/org/zaproxy/zap/extension/ascanrulesAlpha/ExampleFileActiveScanner.java

```diff
@@ -63,6 +65,10 @@ public String getName() {
 		return Constant.messages.getString(MESSAGE_PREFIX + "name");
 	}
 	
+	public boolean targets(TechSet technologies) { 
+		return technologies.includes(Tech.Lang.C); 

```

Comment:
Might worth adding a comment explaining what's the purpose of the method (since JavaDoc might not be available in the medium used to read the examples).
Same comment for the other "targets" method.